projects/plugins/crm/tests/php/event-manager/class-event-manager-faker.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Automattic\Jetpack\CRM\Event_Manager\Tests;
+
+class Event_Manager_Faker {
```

Comment:
Why don't we use just one faker for all tests?
This one has some duplication when compared to Automation_Faker.

Reply:
It doesn't have the same expected data structure.

Reply:
I've removed unused fake data copied from the automation tests